pkgs/development/embedded/bootcommander/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchgit, cmake, libopenblt }:
+
+stdenv.mkDerivation rec {
+
+  pname = "bootcommander";
+  version = "011500";
+
+  src = fetchgit {
+    url = "https://github.com/feaser/openblt.git";
+    rev = "openblt_v${version}";
+    sparseCheckout = [ "Host/Source/" ];
+    sha256 = "sha256-cWdc6Imd6IBqYQRNEafp3bDDQLgdpE90fxSkY+vzhAM=";
+  };
+
+  preConfigure = "cd ./Host/Source/BootCommander/";
+
+  installPhase = ''
+    ls
+    cd $NIX_BUILD_TOP/openblt/Host/
```

Comment:
Please use relative directors

Reply:
In this case relative directories don't give  context where files are and it is more difficult to debug and work later on, the relative path should be something like `../../../../`

Reply:
Using NIX_BUILD_TOP should be avoided as much as possible as it doesn't need to be /build, eg on darwin it is something else.